tests/test_neighbor.py

```diff
@@ -389,10 +389,32 @@ def test_NeighborAddRemoveIpv4WithVrf(self, dvs, testlog):
             tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_NEIGHBOR_ENTRY")
             current_neigh_entries_cnt = len(tbl.getKeys())
             dec_neigh_entries_cnt = (old_neigh_entries_cnt - current_neigh_entries_cnt)
-            assert dec_neigh_entries_cnt == 1
-
-
-# Add Dummy always-pass test at end as workaroud
-# for issue when Flaky fail on final test it invokes module tear-down before retrying
-def test_nonflaky_dummy():
-    pass
+            assert dec_neigh_entries_cnt == 1
+
+    def test_FlushResolveNeighborIpv6(self, dvs, testlog):
+        appl_db = swsscommon.DBConnector(swsscommon.APPL_DB, dvs.redis_sock, 0)
+        prod_state_tbl = swsscommon.ProducerStateTable(appl_db, swsscommon.APP_NEIGH_RESOLVE_TABLE_NAME)
+        fvs = swsscommon.FieldValuePairs([("mac", "52:54:00:25:06:E9")])
+
+        prod_state_tbl.set("Vlan2:2000:1::1", fvs)
+        time.sleep(2)
+
+        (exitcode, output) = dvs.runcmd(['sh', '-c', "supervisorctl status nbrmgrd | awk '{print $2}'"])
+        assert output == "RUNNING\n"

```

Comment:
Is it ok if you can also check whether the req has been sent out? Like parsing `ip neigh show` for 2000:1::1. This way we can verify the functionality. It might be in 'failed' state which is ok.

Reply:
The `ip neigh show` command produced the empty output, it seems like because in `test_FlushResolveNeighborIpv6` we don't test the whole flow of flushing the neighbor entry, because it is impossible to do on the VS.